vision/google/cloud/vision/annotations.py

```diff
@@ -21,6 +21,21 @@
 from google.cloud.vision.safe import SafeSearchAnnotation
 
 
+FACE_ANNOTATIONS = 'faceAnnotations'
+IMAGE_PROPERTIES_ANNOTATION = 'imagePropertiesAnnotation'
+SAFE_SEARCH_ANNOTATION = 'safeSearchAnnotation'
```

Comment:
@daspecster I said the map **and** the string constants. `FACE_ANNOTATIONS`, etc. are still public